frontend/public/components/dashboards-page/overview-dashboard/details-card.tsx

```diff
@@ -0,0 +1,125 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+import { connect } from 'react-redux';
+
+import {
+  DashboardCard,
+  DashboardCardBody,
+  DashboardCardHeader,
+  DashboardCardTitle,
+} from '../../dashboard/dashboard-card';
+import { DetailsBody, DetailItem } from '../../dashboard/details-card';
+import { withDashboardResources, DashboardItemProps } from '../with-dashboard-resources';
+import { InfrastructureModel, ClusterVersionModel } from '../../../models';
+import { referenceForModel, K8sResourceKind, getOpenShiftVersion, getK8sGitVersion, getClusterName, ClusterVersionKind } from '../../../module/k8s';
+import { FLAGS } from '../../../const';
+import { featureReducerName, flagPending } from '../../../reducers/features';
+import { RootState } from '../../../redux';
+import { FirehoseResource } from '../../utils';
+
+const getInfrastructurePlatform = (infrastructure: K8sResourceKind): string => _.get(infrastructure, 'status.platform');
+
+const clusterVersionResource: FirehoseResource = {
+  kind: referenceForModel(ClusterVersionModel),
+  namespaced: false,
+  name: 'version',
+  isList: false,
+  prop: 'cv',
+};
+
+const infrastructureResource: FirehoseResource = {
+  kind: referenceForModel(InfrastructureModel),
+  namespaced: false,
+  name: 'cluster',
+  isList: false,
+  prop: 'infrastructure',
+};
+
+const mapStateToProps = (state: RootState) => ({
+  openShiftFlag: state[featureReducerName].get(FLAGS.OPENSHIFT),
```

Comment:
name nit: when the product name as a variable starts with a lowercase letter, it should be `openshift`

```suggestion
  openshiftFlag: state[featureReducerName].get(FLAGS.OPENSHIFT),
```

Reply:
done